packages/perennial-examples/package.json

```diff
@@ -35,7 +37,7 @@
   "license": "APACHE-2.0",
   "dependencies": {
     "@equilibria/perennial": "0.2.1",
-    "@equilibria/root": "0.1.6",
+    "@equilibria/root": "0.2.0",
```

Comment:
nit: do we need this for deployments specifically? otherwise might be easier to keep track of in another PR

Reply:
We need the new version for Perennial Core (crosschain owner) so I just upgraded all the packages to keep them consistent